### ISSUE TITLE
describe how to set up php-fpm error reporting

### ISSUE THREAD

**Issue by jaylinski** [jaylinski/docker-dev-phpstorm#1]
describe how to set up php-fpm error reporting

**Comment by jaylinski** [comment]
Use the following settings in `php.ini`:

```
error_reporting = E_ALL
display_errors = On
log_errors = On
error_log = "/dev/stderr"
```

Tail the logs with `docker-compose logs -f php70`.

**Comment by jaylinski** [comment]
Solved by 2cc984e36cccf76d32d016ff4907aae1f0df2453.
